server/sonar-web/src/main/js/apps/projects/query.ts

```diff
@@ -220,11 +216,11 @@ function convertSize(metric: string, size: number): string {
     case 1:
       return metric + ' < 1000';
     case 2:
-      return metric + ' >= 1000';
+      return `${metric} >= 1000 AND ${metric} < 10000`;
     case 3:
-      return metric + ' >= 10000';
+      return `${metric} >= 10000 AND ${metric} < 100000`;
     case 4:
-      return metric + ' >= 100000';
+      return `${metric} >= 100000 AND ${metric} < 500000`;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a template literal instead of concatenating with a string literal. ([typescript:S3512](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3512))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3141&issues=AWIKRrzD54IFfP4XjNB4&open=AWIKRrzD54IFfP4XjNB4)

[ISSUE_KEY]: AWIKRrzD54IFfP4XjNB4